aria/model/model.py

```diff
@@ -147,20 +82,18 @@ def __init__(self, model_config: ModelConfig):
         self.max_seq_len = model_config.max_seq_len
 
         # Positional embeddings
-        if model_config.yarn_config is not None:
-            # TODO: Need more testing on this;
-            cfg = model_config.yarn_config
-            self.rotary_emb = DynamicYaRNScaledRotaryEmbedding(
-                self.d_head,
-                max_position_embeddings=round(self.max_seq_len * cfg.scale),
-                original_max_position_embeddings=self.max_seq_len,
-                beta_fast=cfg.beta_fast,
-                beta_slow=cfg.beta_slow,
-                base=cfg.base,
-                mscale_coeff=cfg.mscale_coeff,
-            )
-        else:
-            self.rotary_emb = RotaryEmbedding(self.d_head)
+        cfg = model_config.yarn_config or YaRNConfig()
+        self.rotary_emb = YaRNScaledRotaryEmbedding(
```

Comment:
Are YaRN rotary embeddings always being used? What about when we are pre-training? Does the default YaRNConfig() encapsulate the situation where YaRNScaledRotaryEmbedding is the same as normal rotary embeddings?

Reply:
YaRN embedding is always used. The default is equivalent to the usual RoPE (scale=1.0)